### PR TITLE
Migrate apps secrets from KES to ESO

### PR DESCRIPTION
**Pull request by steckhelena**

### CODE REVIEW

**Review by esdrasbrz — CHANGES_REQUESTED** [review]
You need also to bump manually the Chart.yaml version

File: charts/service/templates/06_secret.yaml [hunk 1]
```diff
@@ -9,18 +9,22 @@
 {{- /* */}}
 {{ $envName := $environment.name | default "prod" }}
 {{ $secretName := printf "%s--%s--%s" $envName $.Values.name $environment.secret.name }}
-apiVersion: kubernetes-client.io/v1
+apiVersion: external-secrets.io/v1beta1
 kind: ExternalSecret
 metadata:
   name: {{ $secretName }}
+  namespace: {{ $namespace }}
```
Comment (esdrasbrz) [inline charts/service/templates/06_secret.yaml:16]: You need to define the namespace in this file too

**Review by esdrasbrz — CHANGES_REQUESTED** [review]

File: charts/service/templates/06_secret.yaml (hunk 1 above)
Comment (steckhelena) [reply charts/service/templates/06_secret.yaml:16]: It is already defined [here](https://github.com/getcircuit/service-chart/pull/15/files/74e9490a31ec09da1125d65bb15d0ea385cbcbf0#diff-1dac3cf55864ebee073271e7ad240cc974fd0991bb622f75c20876173ec6d997R1)

Comment (esdrasbrz) [reply charts/service/templates/06_secret.yaml:16]: Oops

**Review by esdrasbrz — APPROVED** [review]

**Pull request merged**